src/Schedule.cpp

```diff
@@ -36,8 +42,65 @@ EXPORT void destroy<ScheduleContents>(const ScheduleContents *p) {
     delete p;
 }
 
+namespace {
+
+// Deep-copy data from 'src' to 'dst'. Ignore copying 'specialization' over.
```

Comment:
I was originally worried about the recursion, but on further inspection, it was fine. Moved the specialization deep-copy inside the deep_copy_schedule_contents_helper.